src/builder.ts

```diff
@@ -366,6 +381,15 @@ export class SchemaBuilder {
     return this.config;
   }
 
+  // /**
+  //  * Find out if a type has been defined or not, and if it has been, return it.
+  //  * This is useful, for example, for plugins that want to somehow adapt to the
+  //  * user's type defs.
+  //  */
+  // getNamedType(name: string): null | GraphQLNamedType {
+  //   return this.finalTypeMap[name] || null;
+  // }
```

Comment:
Remove before PR merge. This was me working through approach where plugins use builder.